### PR TITLE
Javascript assignment

### PR DESCRIPTION
**Pull request by NikhithaReddy-26**
Q1. Write a program to demonstrate how a function can be passed as a parameter to another function.
Q2. An arrow function takes two arguments firstName and lastName and returns a 2 letter string that represents the first letter of both the arguments. For the arguments Roger and Waters, the function returns ‘RW’. Write this function

### CODE REVIEW

File: code2.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const getInitials = (firstName, lastName) => {
+    return `${firstName.charAt(0)}${lastName.charAt(0)}`;
```
Comment (msaisrinivas) [inline code2.js]: Change the initials into uppercase.

File: assignment.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
+#include <iostream>
```
Comment (msaisrinivas) [inline assignment.cpp]: remove the already pushed file from PR

**Review by msaisrinivas — CHANGES_REQUESTED** [review]

File: assignment.cpp (hunk 2 above)
Comment (NikhithaReddy-26) [reply assignment.cpp]: Done

File: code2.js (hunk 1 above)
Comment (NikhithaReddy-26) [reply code2.js]: done

**Review by msaisrinivas — APPROVED** [review]

**Pull request merged**